src/candidates/profile/chat/shared/navigation/ChatNavigation.vue

```diff
@@ -69,7 +68,28 @@ export default {
   methods: {
     ...mapActions({
       selectContact: 'chat/selectContact'
-    })
+    }),
+    filterItemsMatchingSearch (list) {
+      if (!this.searchText) {
+        return list;
+      }
+
+      const includesIgnoreCase = (str, suffix) =>
+        str.toLowerCase().includes(suffix.toLowerCase());
+
+      return list.filter((item) => {
+        const company = item.company.name;
+        const rep = item.latestRepresentative;
+
+        const matchesCompany = includesIgnoreCase(company, this.searchText);
+        const matchesRepresentative = includesIgnoreCase(rep, this.searchText);
```

Comment:
My hands were all kinds of itchy to replace `includesIgnoreCase` with something like
```
const matchesSearchText = (str) =>
  str.toLowerCase().includes(this.searchText.toLowerCase());
```

Then this code would look as follows:
```
const matchesCompany = matchesSearchText(company);
const matchesRepresentative = matchesSearchText(rep);
```

However, I ultimately decided that the extra-fanciness would come at the cost of the straightforwardness of the code.